src/mongo/nodes.ts

```diff
@@ -211,11 +214,17 @@ export class MongoCollectionNode implements INode {
 		};
 	}
 
-	readonly command: Command = {
-		command: 'cosmosDB.openMongoCollection',
-		arguments: [this],
-		title: ''
-	};
+	readonly collapsibleState = vscode.TreeItemCollapsibleState.Collapsed;
+
+	async getChildren(): Promise<INode[]> {
+		const command = MongoCommands.getCommand(`db.${this.label}.find()`);
+		const parentNode = this;
+		let result = await MongoCommands.executeCommand(command, this.db).then(documents => JSON.parse(documents));
+		if (!Array.isArray(result)) {
+			result = [result];
+		}
+		return result.map(document => new MongoDocumentNode(document._id, parentNode, document));
```

Comment:
nit: I prefer not define single-use variables. In this case you could just use `this` instead of `parentNode`